website/docs/docs/collaborate/govern/model-versions.md

```diff
@@ -60,51 +141,178 @@ models:
 
-By convention, dbt will expect those two models to be defined in files named `dim_customers_v1.sql` and `dim_customers_v2.sql`. It is possible to override this by setting a `defined_in` property.
+**Which version is "latest"?** If not specified explicitly, the `latest_version` would be `2` (numerically greatest). In this case, `v1` is specified to still be the latest; `v2` is a prerelease in early development. When we're ready to roll out `v2` to everyone by default, we would bump the `latest_version` to `2`, or remove it from the specification.
 
-You can reconfigure each version independently. For example, if you wanted `dim_customers.v1` to continue populating the database table named `dim_customers` (its original name), you could use the `defined_in` configuration:
+### Configuring versioned models
+
+You can reconfigure each version independently. For example, you could materialize `v2` as a table and `v1` as a view:
+
+<File name="models/schema.yml">
+
+```yml
+versions:
+  - v: 2
+    config:
+      materialized: table
+  - v: 1
+    config:
+      materialized: view
+```
+
+</File>
+
+Like with all config inheritance, any configs set _within_ the versioned model's definition (`.sql` or `.py` file) will take precedence over the configs set in yaml.
+
+### Configuring database location with `alias`
+
+Following the example, let's say you wanted `dim_customers.v1` to continue populating the database table named `dim_customers`. That's what the table was named previously, and you may have several other dashboards or tools expecting to read its data from `<dbname>.<schemaname>.dim_customers`.
+
+You could use the `alias` configuration:
+
+<File name="models/schema.yml">
 
 ```yml
       - v: 1
-        defined_in: dim_customers # keep original relation name
+        config:
+          alias: dim_customers   # keep v1 in its original database location
 ```
 
+</File>
+
+Or, you could do one better: Define a post-hook to create a view named `dim_customers`, which always points to the latest version of the `dim_customers` model. You can find logic for just such a hook in [this gist](https://gist.github.com/jtcohen6/68220cd76b0bde088d3439664ccfb013/edit). Then, you can implement this for all versioned models in your project:
+
+<File name="dbt_project.yml">
+
+```sql
+{% macro create_latest_version_view() %}
+
+    {% if model.get('version') and model.get('version') == model.get('latest_version') %}
+
+        {% set new_relation = api.Relation.create(
+            database = this.database,
+            schema = this.schema,
+            identifier = model['name']
+        ) %}
+        
+        {% set existing_relation = load_relation(new_relation) %}
+        {{ drop_relation_if_exists(existing_relation) }}
```

Comment:
Since the whole point is that this view should be live-queryable from a BI tool... I don't think we want to be dropping it outside of a transaction / atomic operation.

Not sure if there's a handy cross-db way to do this, given our existing constructs, so this might take some fudging.